hr_attendance_calendar/models/hr_attendance_day.py

```diff
@@ -181,7 +184,26 @@ def _compute_break_total(self):
     @api.depends('worked_hours', 'due_hours')
     def _compute_extra_hours(self):
         for att_day in self.filtered('worked_hours'):
-            att_day.extra_hours = att_day.worked_hours - att_day.due_hours
+            extra_hours = att_day.worked_hours - att_day.due_hours
+            max_extra_hours = float(self.env['ir.config_parameter'].get_param(
+                'hr_attendance_calendar.max_extra_hours'))
+
+            if extra_hours > max_extra_hours:
+                att_day.extra_hours = max_extra_hours
+            else:
+                att_day.extra_hours = extra_hours
+
+    @api.multi
+    @api.depends('extra_hours')
+    def _compute_extra_hours_lost(self):
+        for att_day in self.filtered('worked_hours'):
+            extra_hours = att_day.worked_hours - att_day.due_hours
```

Comment:
already compute in att_day.extra_hours

Reply:
yes, but extra_hours can already have the value of max_extra_hours at this state of the execution, so it's necessary to recompute this value